src/lightning/pytorch/loggers/mlflow.py

```diff
@@ -36,9 +36,7 @@
 
 log = logging.getLogger(__name__)
 LOCAL_FILE_URI_PREFIX = "file:"
-_MLFLOW_FULL_AVAILABLE = RequirementCache("mlflow>=1.0.0")
-_MLFLOW_SKINNY_AVAILABLE = RequirementCache("mlflow-skinny>=1.0.0")
```

Comment:
There's no need to check for skinny explicitly anymore, because both the full and skinny versions are imported with `mlflow`